Apollo.xcodeproj/project.pbxproj

```diff
@@ -248,6 +250,9 @@
 		DED46051261CEAD20086EF63 /* StarWarsAPI.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 9FCE2CFA1E6C213D00E34457 /* StarWarsAPI.framework */; };
 		E616B6D126C3335600DB049E /* ExecutionTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = E616B6D026C3335600DB049E /* ExecutionTests.swift */; };
 		E61DD76526D60C1800C41614 /* SQLiteDotSwiftDatabaseBehaviorTests.swift in Sources */ = {isa = PBXBuildFile; fileRef = E61DD76426D60C1800C41614 /* SQLiteDotSwiftDatabaseBehaviorTests.swift */; };
+		E69C255026EC2A9D00E9495D /* ApolloCodegenLib.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 9B7B6F47233C26D100F32205 /* ApolloCodegenLib.framework */; };
```

Comment:
`ApolloCodegenLib` was linked to the `ApolloPerformanceTests` target because `MockNetworkSession` is part of the `ApolloTestSupport` bundle which is a dependency of `ApolloPerformanceTests`.

Reply:
I think it should be in `ApolloTestSupport`.

Reply:
It is in the `ApolloCodegenTestSupport` target.